account_payment_term_extension/models/account_payment_term.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
```

Comment:
You can remove the utf-8 lines from the top in Python3. Just safe the file in utf8 format.